### PR TITLE
Remove modified method SDDP.write_cuts_to_file

### LINKED ISSUE #7: Audit sddp_modifications.jl
Much of the code in https://github.com/EPOC-NZ/JADE.jl/blob/main/src/sddp_modifications.jl should be moved into SDDP.jl with appropriate modifications. Ideally, we won't have any SDDP.jl extensions in JADE.jl.

### PR DESCRIPTION
**Pull request by odow**
Instead of writing a completely new `write_cuts_to_file` method, it's simpler to leverage the <s>existing</s> new SDDP implementation<s>, and then post-hoc rename the nodes</s>.

<s>Part of</s> Closes #7.

- [x] This needs tests before merging

### CODE REVIEW

**Comment by odow** [comment]
I found a better way to support this in SDDP.jl.

**Comment by odow** [comment]
Tests will fail until https://github.com/odow/SDDP.jl/pull/567 is merged and a new release is tagged.

File: test/test_cases.jl [hunk 1]
```diff
@@ -235,6 +235,31 @@ function test_case_7()
...
+    # Now build a finite horizon policy using the EOH cuts
+    data.steady_state = false
+    options.write_eohcuts = false
+    options.reset_starting_levels = true
+    data.number_of_wks = 10
+    options.eoh_cutfile = "2008_steady_state"
+    model_finite = create_JADE_model(data, HiGHS.Optimizer);
+    optimize_policy!(model_finite, options)
+    @test length(model_finite.sddpm.nodes) == 10
+    @test model_finite.sddpm.most_recent_training_results.status == :iteration_limit
```
Comment (odow) [inline test/test_cases.jl:270]: @adow031 I'm going to need your input on this one. Is this how I create a finite horizon policy?

Comment (adow031) [reply test/test_cases.jl:270]: Yes; this looks correct.

Comment (odow) [reply test/test_cases.jl:270]: Okay. I've fleshed it out even more, adding a comparison with the terminal water values as well.

**Pull request merged**